pinot-spi/src/main/java/org/apache/pinot/spi/utils/validator/JsonValidator.java

```diff
@@ -0,0 +1,13 @@
+package org.apache.pinot.spi.utils.validator;
+
+import java.io.Serializable;
+import java.util.List;
+
+
+public interface JsonValidator extends Serializable {
```

Comment:
Maybe use a better name for this interface ? Do we expect this to be a common interface for any kind of json ? Or just limited to config (table or schema) ? 

If its just config - we could also say PinotConfigValidator or something ?